src/completion.js

```diff
@@ -1,8 +1,8 @@
 "use strict"
 
 const vscode = require('vscode')
-const common = require('./common')
-const { attributes, blocks, functions, variables, DEFAULT_LABEL } = require('./schema')
+const common = require('common')
+const { attributes, blocks, functions, variables, DEFAULT_LABEL } = require('schema')
```

Comment:
This breaks it for me.

Reply:
Fixed in https://github.com/avenga/couper-vscode/releases/tag/v1.0.1
thanks.